homework-assignments-1/task_4.ts

```diff
@@ -0,0 +1,170 @@
+class Product {
+    name:string;
+    price:number;
+    stock:number;
+    constructor(name:string, price:number, stock:number) {
+        this.name = name;
+        this.price = price;
+        this.stock = stock;
+    }
+
+    addStock(quantity:number):void{
+        this.stock += quantity;
+        console.log(`${quantity} items added. New stock: ${this.stock}`);
+    };
+    removeStock(quantity:number):void{
+        this.stock -= quantity;
```

Comment:
Don't forget to check quantity removing from stck must be less than stock available